ldms/src/sampler/rdc_sampler/rdc.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I am not sure why this script exists. All it does is output a static rdc.conf file. We should probably just include an rdc.conf example file rather than having a script. (Update README if this is changed).

Reply:
agree